src/doc/unstable-book/src/language-features/member-constraints.md

```diff
@@ -1,29 +0,0 @@
-# `member_constraints`
```

Comment:
Is this worth adding to the reference? Not sure where it would go though.

Reply:
I considered this, but I don't think so. I think the reference should eventually be extended with a more thorough description of the formal type system, but this is really an implementation detail at present. There is good coverage in the rustc-dev-guide.